.github/workflows/main.yml

```diff
@@ -255,7 +255,8 @@ jobs:
         exit 1
   linux-musl:
     runs-on: ubuntu-latest
-    container: alpine
+    container:
```

Comment:
No, in that page:

> When you only specify a container image, you can omit the `image` keyword.
```
jobs:
  my_job:
    container: node:10.16-jessie
```

I'll drop this commit

Reply:
Okay, I used the VS Code extension which apparently has an outdated/incomplete JSON schema.